jsonrpc/src/schemas/errors.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ContentNotFoundError": {
+    "code": 32001,
+    "message": "RecursiveFindContent failed: Unable to locate content on the network.",
+    "data": {
+      "$ref": "#/components/schema/traceResultObject"
+    }
```

Comment:
My thinking was... The spec defines the `"data"` field as optional. So for non-traced requests, the client implementation can choose whether or not to include the trace. And then for traced requests, the client should already have the trace and include it in the error message.

Reply:
Conceptually, I'm not sure if I really like this idea of having the same object returned, but once as success and once as an error.

I think it also means that glados needs to be adapted to interpret these error codes?

Reply:
@mrferris ping for glados. I'll continue to fiddle with the actual json so that it displays properly. 

I'm not sure if I follow: ` but once as success and once as an error.` Just to make sure we're on the same page, these are the 4 relevant outcomes as I understand... (but `"traceObjectResult"` is the actual trace object)

For successful RFC:
```json
{
  "result": {
    "content": "0x...",
    "utpTransfer": false
  }
}
```

For successful RFC with trace enabled
```json
{
  "result": {
    "content": "0x...",
    "utpTransfer": true,
    "trace": "traceObjectResult"
  }
}
```

For failed RFC:
```json
{
  "error": {
    "code": -39001,
    "message": "Unable to locate content on the network."
  }
}
```

For failed RFC with trace enabled.
```json
{
  "error": {
    "code": -39001,
    "message": "Unable to locate content on the network.",
    "data": "traceObjectResult"
  }
}
```

Reply:
> but once as success and once as an error

Yeah, I meant here that the `traceObjectResult`, which can be considered as a result, is in one case passed in the "result" json response and once in the "error" json response. At first sight that looked conceptually strange to me, but looking at your provided json extracts now, it does make it more consistent with the trace version. So I'm fine with using the error route too.

If the `content` field and `utpTransfer` field would be part of the `traceObjectResult`, it would be different and I'd prefer not to use the error to pass that object along.

In the case of glados, I assume that currently the `content` field is checked first or perhaps the `receivedFrom` field of the `traceObjectResult` is checked first to verify the case of content not found. With this change, it would simply have to parse the error json and check the error code + use the trace object.